supervisor/supervisor.go

```diff
@@ -125,15 +125,23 @@ func (s *Supervisor) ScheduleAdhoc(a *db.Task) {
 		// expect a JobUUID to move to the schedq Immediately
 		for _, job := range s.jobq {
 			if !uuid.Equal(job.UUID, a.JobUUID) {
+
 				continue
 			}
 
 			log.Infof("scheduling immediate (ad hoc) execution of job %s [%s]", job.Name, job.UUID)
 			task, err := s.Database.CreateBackupTask(a.Owner, job)
 			if err != nil {
 				log.Errorf("job -> task conversion / database update failed: %s", err)
+				if a.TaskUUIDChan != nil {
+					a.TaskUUIDChan <- ""
```

Comment:
should we pass the error here back through the channel, so the API can return more meaningful error messages on failure? is that more work than it's worth?